sklearn/metrics/_plot/tests/test_roc_curve_display.py

```diff
@@ -193,7 +198,6 @@ def test_roc_curve_chance_level_line(
         assert display.chance_level_.get_linestyle() == "--"
         assert display.chance_level_.get_label() == "Chance level (AUC = 0.5)"
     elif plot_chance_level:
-        assert display.chance_level_.get_label() == chance_level_kw["label"]
```

Comment:
I removed this test as it is redundant with the `assert chance_level_kw["label"] in legend_labels` added further below.